plugins/woocommerce/changelog/43803-fix-42280-change-address-link-in-cart-block

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+- Hide the shipping calculator from the Cart editor when there is no default address and shipping zone.  - Display the change address link when there are multiple shipping zones available along with a fallback shipping zone.
```

Comment:
This changelog message might appear in the release notes. I'd probably shorten it and avoid the use of dashes.